core/watcher/eventsource/package_test.go

```diff
@@ -22,27 +24,50 @@ func TestPackage(t *testing.T) {
 	gc.TestingT(t)
 }
 
+type ImportTest struct{}
+
+var _ = gc.Suite(&ImportTest{})
+
+func (*ImportTest) TestImports(c *gc.C) {
+	found := coretesting.FindJujuCoreImports(c, "github.com/juju/juju/core/watcher/eventsource")
+
+	// This package brings in nothing else from outside juju/juju/core
+	c.Assert(found, jc.SameContents, []string{
+		"core/changestream",
+		"core/database",
+		"core/life",
+		"core/migration",
+		"core/network",
+		"core/resources",
+		"core/secrets",
+		"core/status",
+		"core/watcher",
+		"docker",
```

Comment:
This has me curious. Any idea why docker is needed here? Happy to have a look if need be.

Reply:
I'm not 100% sure, but we do need to remove it.